test/benchmark.cc

```diff
@@ -1008,6 +1299,7 @@ int main(int args, char** argv){
 	fprintf(stdout, "(Note: cdada non-custom containers don't support >256 bytes)\n");
 	fprintf(stdout, "============================================================\n\n");
 
+#if 0
```

Comment:
Dead code?

Reply:
Nope. This was an optimization on my part to run just the stack benchmark. I've removed the following statements.

<pre>#if 0 ... #endif</pre>

Reply:
Done.